docs/cli-arguments.md

```diff
@@ -46,6 +46,7 @@ Usage of ./kube-state-metrics:
       --metric-opt-in-list string                  Comma-separated list of metrics which are opt-in and not enabled by default. This is in addition to the metric allow- and denylists
       --namespaces string                          Comma-separated list of namespaces to be enabled. Defaults to ""
       --namespaces-denylist string                 Comma-separated list of namespaces not to be enabled. If namespaces and namespaces-denylist are both set, only namespaces that are excluded in namespaces-denylist will be used.
+      --nodename string                            Name of the node that contains the kube-state-metrics pod, only available for resources (pod metrics) that support spec.nodeName fieldSelector. Each kube-state-metrics pod will only exposes metrics related to this node.
```

Comment:
To make sure that users don't use this option under normal circumstances, could you perhaps mention that this used for sharding? In a similar way as what is done today for the `--pod` option.
https://github.com/kubernetes/kube-state-metrics/blob/master/docs/cli-arguments.md?plain=1#L50

Also, to make it consistent with the existing `--pod` option I would suggest renaming the nodename option to `--node`.

Reply:
Done.